src/nbl/asset/utils/CDerivativeMapCreator.cpp

```diff
@@ -187,6 +82,9 @@ core::smart_refctd_ptr<ICPUImage> CDerivativeMapCreator::createDerivativeMapFrom
 	state.scratchMemoryByteSize = DerivativeMapFilter::getRequiredScratchByteSize(&state);
 	state.scratchMemory = reinterpret_cast<uint8_t*>(_NBL_ALIGNED_MALLOC(state.scratchMemoryByteSize, _NBL_SIMD_ALIGNMENT));
 
+	if (!DerivativeMapFilter::blit_utils_t:: template computeScaledKernelPhasedLUT<float>(state.scratchMemory + DerivativeMapFilter::getScratchOffset(&state, DerivativeMapFilter::ESU_SCALED_KERNEL_PHASED_LUT), state.inExtentLayerCount, state.outExtentLayerCount, state.inImage->getCreationParameters().type, convolutionKernels))
+		return nullptr;
```

Comment:
isn't there a cleaner API to call to populate the LUT?

I mean most of the arguments you're passing are already in your `state`

Reply:
I addded one as part of #499

Reply:
Yep, incorporated it.
Resolved.